tools/lsp/properties.rs

```diff
@@ -323,11 +324,16 @@ fn find_expression_range(
             }
         }
     }
-    expression_value.map(|e| DefinitionInformation {
-        property_definition_range: offset_to_position.map_range(property_definition_range),
-        delete_range: offset_to_position.map_range(delete_range),
-        expression_range: offset_to_position.map_range(expression_range),
-        expression_value: e,
+
+    (delete_range.is_some()
+        && expression_range.is_some()
+        && expression_value.is_some()
+        && property_definition_range.is_some())
+    .then(|| DefinitionInformation {
+        property_definition_range: property_definition_range.expect("was some"),
+        delete_range: delete_range.expect("was some"),
+        expression_range: expression_range.expect("was some"),
+        expression_value: expression_value.expect("was some"),
     })
```

Comment:
This is_some / expect("was some")  is horrible.  in this case you can use the ? operator:
```rust
DefinitionInformation {
        property_definition_range: property_definition_range?,
        delete_range: delete_range?,
        expression_range: expression_range?,
        expression_value: expression_value?,
    }
```